types/stylelint/index.d.ts

```diff
@@ -616,6 +634,7 @@ declare namespace stylelint {
 		 * Optional severity override for the problem.
 		 */
 		severity?: RuleSeverity;
+		unfixable?: boolean;
```

Comment:
[ask] To make sure, what case do you assume using `unfixable`?

Reply:
explained in the comments, but I found a better alternative
in short some fixable migrated rules have cases where they can't fix (so you won't have a fix callback)
these cannot be distinguished from unmigrated fixable rules when adding fix data